docker/cypress/Dockerfile

```diff
@@ -4,7 +4,7 @@
 
 FROM --platform=linux/arm64 node:16.13.0-bullseye as builder
 
-ARG VERSION=9.4.1
+ARG VERSION=10.4.0
```

Comment:
I think we can delete this whole file, and the README in this directory, since `docker-compose.cypress-m1.yml` relies on official image now, and this is no longer used? 

Looks like we could probably use same `yml` locally and on circle, except environment variables are handled slightly differently. Maybe easier to keep separate for now.

Reply:
Yeah, I tried to unify the environment variable use between those and couldn’t find anything that worked.

Reply:
I removed those two files, and all tests still work fine.